projects/components/src/dropdown/dropdown.component.ts

```diff
@@ -27,6 +27,10 @@ interface DropdownItem<T extends DropdownItem<T>> {
      * To mark if the {@link #ActionItem.textKey} has to be translated or not
      */
     isTranslatable?: boolean;
+    /**
+     * Css class of the dropdown item. Must be unique among all dropdown items within the dropdown items list
+     */
+    class: string;
```

Comment:
please name this className, class is a reserved word, it's allowed here but people tend to avoid it to minimize confusion

Reply:
I named it to class by following the existing ActionItem interface in vcd ui to avoid a big refactor. I use the class property coming form the action item object in this component. If I change this to className, I have to either change the action item object as well(a big refactor in our repo and also vcd ui repo) or do some work inside this component top convert class -> className.

Reply:
OK